internal/commands/ostest/test_execution.go

```diff
@@ -63,66 +114,85 @@ func RunTest(
 		return nil, nil, fmt.Errorf("test completed but no result was returned")
 	}
 
+	if finalResult.GetExecutionState() == testapi.Errored {
```

Comment:
Does this result in an exit code 2?

Reply:
Update: LGTM with comments indicating how this sets up the necessary outputs for the GAF to effect the desired exit code.